### PR TITLE
Add trial for reviewing

### PR DESCRIPTION
**Pull request by Fsher07**
I will try to review my own code

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+#title{
+    text-align: center;
+}
+#description{
+    text-align: center;
+}
+button{
+    
+    margin-top: 16px;
```
Comment (Fsher07) [inline style.css:17]: Kindly, try to not leave the unnecessary empty line in your coding rows.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        <select name="regions" id="reg">
+            <option value="North_America">North America</option>
+            <option value="Sout_America">South America</option>
+            <option value="Asia">Asia</option>
+            <option value="Europe">Europe</option>
+            <option value="Africa">Africa</option>
+            <option value="Australia">Australia</option>
+            <option value="Antarctica">Antarctica</option>
+
+        </select><br>
```
Comment (Fsher07) [inline index.html:30]: Please, try to be careful with blank lines and indentations. For readability, add two spaces of indentation. Do not use the tab key or you can change the tab key to make two spaces, not four 😊 .

File: style.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    background-color: royalblue;
+    border-radius: 10px;
+    width: 600px;
+    height: 100px;
+}
+#comment-section{
+    height: 400px;
+}
```
Comment (Fsher07) [inline style.css:25]: Good job so far!  Lastly, the stylelint and webhint linters are not passing, so please fix the problems before asking for another review.

**Review by mahbubul14 — APPROVED** [review]
# Status: Project Approved :trophy: 

Hi @Fsher07 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :+1: 
Congratulations! 🎉 

:heavy_check_mark: Your comments are clear and easy to understand

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**